app/views/auctions/labels/_time_remaining.html.erb

```diff
@@ -0,0 +1,7 @@
+<div class="issue-bid-deadline">
+  <% if auction.over? or auction.future? %>
```

Comment:
the new refactor has us removing all conditionals from views and putting them in the view models -- can you do that here? If you'd like help, I am more than happy to pair.